docker/dockerfile

```
@@ -0,0 +1,13 @@
+FROM node:iron-slim
+
+WORKDIR /user/app
```

```suggestion
WORKDIR /app
```

how about shorten this path?

---

fixed